mm/2s2h/BenGui/BenMenu.cpp

```diff
@@ -1265,6 +1265,9 @@ void BenMenu::AddEnhancements() {
     AddWidget(path, "Skip Song of Time cutscenes", WIDGET_CVAR_CHECKBOX)
         .CVar("gEnhancements.Songs.SkipSoTCutscenes")
         .Options(CheckboxOptions().Tooltip("Skips the cutscenes when playing any of the Song of Time songs."));
+    AddWidget(path, "Skip Soaring cutscene", WIDGET_CVAR_CHECKBOX)
```

Comment:
For consistency's sake
```suggestion
    AddWidget(path, "Skip Song of Soaring cutscene", WIDGET_CVAR_CHECKBOX)
```

Reply:
I want it to be clear that it's the actual soaring that gets skipped, not the cutscene (if you can call it that) where you learn the song.